MergerService/src/run.cs

```diff
@@ -454,6 +480,19 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils, string? managerCallba
             this._logger.LogDebug($"[{methodName}] end");
         }
 
+        private void UpdateTargetTiles(IData target, List<Tile> tiles, MergeTask task, long overallTileProgressCount, long totalTileCount, ITaskUtils taskUtils)
+        {
+            string methodName = MethodBase.GetCurrentMethod().Name;
+
+            using (this._activitySource.StartActivity("saving tiles"))
+            {
+                this._logger.LogInformation($"[{methodName}] Update {tiles.Count} tiles");
```

Comment:
replcae with this._logger.LogInformation($"[{methodName}] Updating tiles chunk Job: {task.JobId}, Task: {task.Id}"); and remove 2 redundent logs before the function

Reply:
done